src/views/ManageList.jsx

```diff
@@ -1,7 +1,108 @@
+import { useCallback, useState } from 'react';
+import { useStateWithStorage } from '../utils';
+import { addItem } from '../api';
+
+const soonDate = 7;
+const kindOfSoonDate = 14;
+const notSoonDate = 30;
+
 export function ManageList() {
+	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(null);
+	const [itemName, setItemName] = useState('');
+
+	const [listPath, setListPath] = useStateWithStorage(
+		'tcl-shopping-list-path',
+		null,
+	);
+
+	// const listPath = 't4XIww03JAXm1QWr6UPEebbLRl13/first list';
+
+	const handleTextChange = (event) => {
+		setItemName(event.target.value);
+	};
+
+	const handleChange = (event) => {
+		const numberOfDays = parseInt(event.target.value);
+		setDaysUntilNextPurchase(numberOfDays);
+	};
+
```

Comment:
Can these `handleChange` functions have a more descriptive name? something like `handleItemChange` and `handleDaysUntilNextPurchase` or something along those lines can make it clear what these functions are supposed to do